### PR TITLE
Continue rm enzyme, unit tests.

### PR DESCRIPTION
**Pull request by tmashuang**
Convert Confirm Page Container and Welcome components to use testing-library/react, and adds test-ids to elements associated with those tests.

### CODE REVIEW

**Review by darkwing — DISMISSED** [review]

**Review by brad-decker — DISMISSED** [review]

File: ui/pages/first-time-flow/welcome/welcome.test.js [hunk 1]
```diff
@@ -22,15 +16,13 @@ describe('Welcome', () => {
...
 
-    const getStartedButton = wrapper.find(
-      '.btn-primary.first-time-flow__button',
-    );
-    getStartedButton.simulate('click');
+    const getStartedButton = screen.getByTestId('first-time-flow__button');
+
+    fireEvent.click(getStartedButton);
+
+    // getStartedButton.simulate('click');
```
Comment (digiwand) [inline ui/pages/first-time-flow/welcome/welcome.test.js:25]:
```suggestion
```

File: ui/pages/first-time-flow/welcome/welcome.test.js [hunk 2]
```diff
@@ -45,15 +37,12 @@ describe('Welcome', () => {
...
-    );
+    renderWithProvider(<Welcome.WrappedComponent {...props} />);
 
-    const getStartedButton = wrapper.find(
-      '.btn-primary.first-time-flow__button',
-    );
-    getStartedButton.simulate('click');
+    const getStartedButton = screen.getByTestId('first-time-flow__button');
+
+    // getStartedButton.simulate('click');
```
Comment (digiwand) [inline ui/pages/first-time-flow/welcome/welcome.test.js:44]:
```suggestion
```

**Review by digiwand — DISMISSED** [review]

File: ui/components/app/confirm-page-container/confirm-page-container-container.test.js [hunk 3]
```diff
@@ -60,119 +28,127 @@ describe('Confirm Page Container Container Test', () => {
...
+
+    it('should render a confirm page container component', () => {
+      const pageContainer = screen.queryByTestId('page-container');
+      expect(pageContainer).toBeInTheDocument();
+    });
+
+    it('should render navigation', () => {
+      const navigationContainer = screen.queryByTestId('navigation-container');
+      expect(navigationContainer).toBeInTheDocument();
+      // expect(wrapper.find(ConfirmPageContainerNavigation)).toHaveLength(1);
```
Comment (digiwand) [inline ui/components/app/confirm-page-container/confirm-page-container-container.test.js:80]:
```suggestion
```

File: ui/pages/first-time-flow/welcome/welcome.test.js (hunk 2 above)
Comment (digiwand) [reply ui/pages/first-time-flow/welcome/welcome.test.js:43]: To confirm, we are keeping this comment?

Comment (digiwand) [reply ui/pages/first-time-flow/welcome/welcome.test.js:43]: To confirm, we are keeping this comment? @tmashuang

Comment (tmashuang) [reply ui/pages/first-time-flow/welcome/welcome.test.js:43]: Hmm I thought this got resolved in [9624b17](https://github.com/MetaMask/metamask-extension/pull/15730/commits/9624b17d190c68f9ef77cff0ac97cce8c6e6198e), will resolve properly.

**Review by digiwand — APPROVED** [review]

**Review by brad-decker — APPROVED** [review]

**Pull request merged**